doc/database-model.md

```diff
@@ -0,0 +1,61 @@
+
+# Database Entity Relationship Diagram
+
+```mermaid
+erDiagram
+    USER {
```

Comment:
add email and password for login without social networks

Reply:
done